### ISSUE TITLE
Create models in DB

### ISSUE THREAD

**Issue by DelfiSamperi** [DelfiSamperi/videogamesApp#4]
Create models in Database for videogames and genres

**Comment by LucasFeli** [comment]
Hi! Can I use MongoDB and Mongoose? If that's the case, can I work on this issue? It's one of my first contributions. ty

**Comment by DelfiSamperi** [comment]
Yes, you can! I'm working with PostgreSQL but I'm open to new technologies, here are the specifications for each model:
MODEL 1 | VIDEOGAMES: 
- ID (different from the ones from the API).
-  Name. \*
-  Description. \*
-  Platforms. \*
-  Image. \*
-  Release date. \*
-  Rating. \*

Genres:

**📍 MODEL 2 | Genres**
-  ID. \*
-  Name. \*

They  have a many to many relationship in Postgres, if not available in MongoDB just dismiss it.

**Comment by LucasFeli** [comment]
Perfect , i'm start to work on it
